<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Contact List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
	outline: none !important; 
}

QLabel {
	font-size: 16px;
	color: #333;
}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QStackedWidget QWidget#homePage, QStackedWidget QWidget#listPage, QStackedWidget QWidget#detailPage, QStackedWidget QWidget#aboutPage {
background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 #acb6e5,  stop:1 #86fde8);
}</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="homePage">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>800</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SVN-Cookies</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 48px;
color: #2d1299;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Digital Contact List</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>220</width>
        <height>240</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 #2d1299,  stop:1 #2980b9);
	padding: 10px 8px 12px 8px;
	font-size: 24px;
	color: #fff;
	border: none;
	border-radius: 15px;
}

QPushButton:hover {
	background: qlineargradient( x1:0 y1:0, x2:1 y2:0, stop:0 #8e2de2,  stop:1 #4a00e0);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <widget class="QPushButton" name="contactListBtn">
         <property name="font">
          <font>
           <family>SVN-Cookies</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Contact List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="aboutUsBtn">
         <property name="font">
          <font>
           <family>SVN-Cookies</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>About Us</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitBtn">
         <property name="font">
          <font>
           <family>SVN-Cookies</family>
           <pointsize>-1</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>160</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>assets/icons/contact.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="listPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>800</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SVN-Cookies</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
color: #2d1299;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Contact List</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="listWidget" native="true">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>99</y>
        <width>600</width>
        <height>420</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#listWidget {
	border-radius: 15px;
}

QWidget {
	background-color: #fff;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="styleSheet">
          <string notr="true">font-size: 16px;
color: #333;
font-weight: bold;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,9">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>7</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>ID</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="listScrollWrapper">
         <property name="styleSheet">
          <string notr="true">QScrollArea {
	border: none;
	border-radius: 20px;
}

QScrollBar:vertical {              
	/*border: 1px solid #999999;*/
	border-radius: 5px;
    background: transparent;
    width: 12px;    
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #f27d85;
	border-radius: 5px;
}

QScrollBar::add-line:vertical {
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
    height: 0 px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="listScrollArea">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>570</width>
            <height>364</height>
           </rect>
          </property>
          <widget class="QFrame" name="frame_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>531</width>
             <height>81</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame#frame_4 {
	border-bottom: 1px solid #ccc;
}</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,8">
            <property name="sizeConstraint">
             <enum>QLayout::SetFixedSize</enum>
            </property>
            <property name="leftMargin">
             <number>11</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>8</number>
            </property>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="text">
               <string>C100000</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <property name="minimumSize">
               <size>
                <width>383</width>
                <height>71</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="maximumSize">
                  <size>
                   <width>64</width>
                   <height>64</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap>alexis_barnett_sm.png</pixmap>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="minimumSize">
                  <size>
                   <width>318</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Alexandir gaga ehehehehe</string>
                 </property>
                 <property name="wordWrap">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>140</y>
        <width>570</width>
        <height>1</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top: 2px solid #333;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="listBackBtn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>535</y>
        <width>200</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SVN-Cookies</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
font-size: 20px;
color: #2d1299;
background-color: none;
border: none;
}

QPushButton:hover {
color: #8e2de2;
}</string>
      </property>
      <property name="text">
       <string>Back to Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="detailPage">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="detailBackBtn">
      <property name="geometry">
       <rect>
        <x>289</x>
        <y>532</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SVN-Cookies</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;
color: #2d1299;
background-color: none;
border: none;</string>
      </property>
      <property name="text">
       <string>Back to Contact List</string>
      </property>
     </widget>
     <widget class="QWidget" name="detailWrapper" native="true">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>57</y>
        <width>560</width>
        <height>461</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#detailWrapper {
	background-color: #fff;
	border-radius: 15px;
}</string>
      </property>
      <widget class="QLabel" name="detailImg">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>20</y>
         <width>260</width>
         <height>260</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #ccc;
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>alexis_barnett.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="detailName">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>297</y>
         <width>560</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 28px;
color: #2d1299;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Alexandir gaga ehehehehe</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>344</y>
         <width>460</width>
         <height>48</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	font-size: 16px;
	color: #333;
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>assets/icons/phone.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="detailPhone">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;</string>
          </property>
          <property name="text">
           <string>01234567890</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="detailMail">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;</string>
          </property>
          <property name="text">
           <string>aaaaaaacdef@gmail.com</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>assets/icons/email.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_5" native="true">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>280</y>
         <width>260</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 16px;</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;</string>
          </property>
          <property name="text">
           <string>ID:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="detailID">
          <property name="text">
           <string>C1</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="detailBio">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>400</y>
         <width>460</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Hello motherfucker, hehehe hahaha, one two three. ABC DEF XGHSAD ASDA DASD asd asdas qwe q asdas da</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="aboutPage">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>800</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>SVN-Cookies</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 30px;
color: #2d1299;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>About Us</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#listWidget {
	background-color: #fff;
	border-radius: 15px;
}</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
